cucumber/features/steps/shared.rb

```diff
@@ -8,6 +8,22 @@ def get_http_route(path)
       body = http({:method => "GET", :path => path}, {})
       response_body_to_hash(body)
     end
+
+    def switch_to_second_tab
+      xcode = RunLoop::Xcode.new
+      if xcode.version_gte_110?
+        # Separate case for Xcode 11 beta since it doesn't switch to the second tab sometimes
+        # retry_count = 15 minutes to make sure that simulator has stabilized
+        retry_count = 900
```

Comment:
Wow.  That is a long time to retry.  Don't change.

Here is good place where this test should just fail when the Xcode 11 version is out of beta.

Not blocking. Nothing to do.